src/pages/projectSettings/tabs/GithubCommitQueueTab/getFormSchema.tsx

```diff
@@ -200,6 +200,11 @@ export const getFormSchema = (
                 repoData?.commitQueue?.requireSigned
               ),
             },
+            requiredApprovalCount: {
+              type: ["number", "null"],
+              title: "Require one or more approvals on Pull Request Merges",
```

Comment:
Can we call this "Required Number of Approvals on Pull Request Merges" instead?

Reply:
done

Reply:
Wording looks good to me, not planning to review whole PR unless needed beyond this